### ISSUE TITLE
[1.7.10]IllegalAccessError on placing a TE Tesseract

### ISSUE THREAD

**Issue by picatrix1899** [SleepyTrousers/EnderIO#4307]
#### Issue Description:
Hello,
when I place a tesseract in the world the game crashes with an IllegalAccessError. If not on placing then on rightclicking. the issue was found in my modpack but could be tracked down to EnderCore, EnderIO, CoFHCore, Thermal Foundation and Thermal Expansion. I used in my modpack another vesion but could reproduce the issue with the latest beta and stable build for 1.7.10.

Is this a known bug with a fix? I've searched a bit but didn't found this problem.

**EDIT:**
If you join a world where you placed a tesseract and destroy it it will crash too. but the rightclick works correctly.

#### What happens:
minecraft crashes.

#### What you expected to happen:
no crash.

#### Steps to reproduce:

1. generate new world
2. place Tesseract from TE.
3. - crash -

1. load a world with a tesseract in it
2. destroy the teseract
3. - crash -

____
#### Affected Versions (Do *not* use "latest"):

- EnderIO: 1.7.10-2.2.8.381
- EnderIO: 1.7.10-2.3.0-429_beta
- EnderCore: 1.7.10-0.2.0.39_beta
- CoFHCore: [1.7.10]3.1.4-329
- Thermal Foundation: [1.7.10]1.2.6-118 
- Thermal Expansion: [1.7.10]4.1.5-248
- Minecraft: 1.7.10
- Forge: 10.13.4.1614

#### Your most recent log file where the issue was present: 

https://pastebin.com/M8k56afM

**Comment by HenryLoenwind** [comment]
-> https://github.com/CoFH/Feedback

**Comment by picatrix1899** [comment]
Hello again,
I've posted an issue at CoFH. KingLemming told me:

> Caused by a mod including an invalid version of the CoFH API. Quite possibly EnderIO. You'll have to perform jar surgery.

Issue at CoFH:
https://github.com/CoFH/Feedback/issues/470

I investigated further including checking sourcecodes. But I'm not really good in modding so I barly know whats going on.

The result of the investigation is that the enderIO version 2.2.8.381 does actually work I've just forgotten to remove EnderCore.
But EnderCore itself causes the problem. I tested a "halfway clean" setup only with Liteloader, CoFHCore, Thermal Foundation, Thermal Expansion and EnderCore. I used very early versions and late versions of EnderCore but not one worked. I can only ask you because there is only one version of CoFHCore for 1.7.10 out there available. I would remove EnderCore from the modpack but then I have to remove WailaPlugins too.

EnderCore is a long while out there and no one ever had this problem? what am I doing wrong?

tested versions:
- EnderCore: 1.7.10-0.2.0.39_beta
- CoFHCore: [1.7.10]3.1.4-329
- Thermal Foundation: [1.7.10]1.2.6-118
- Thermal Expansion: [1.7.10]4.1.5-248
- Minecraft: 1.7.10
- Forge: 10.13.4.1614

latest crashlog:
https://pastebin.com/qbdRyLh5
